TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot.java

```diff
@@ -2,18 +2,22 @@
 
 import com.qualcomm.robotcore.eventloop.opmode.OpMode;
 import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
-import com.qualcomm.robotcore.hardware.HardwareMap;
 
 @TeleOp
 public class Robot extends OpMode {
 
+    servoClaw servoClaw = new servoClaw();
+    servoWrist servoWrist = new servoWrist();
+
     @Override
     public void init() {
-
+        servoClaw.init(hardwareMap);
+        servoWrist.init(hardwareMap);
     }
 
     @Override
     public void loop() {
-
```

Comment:
I dont know if the control scheme has been finalized yet, if so the controls are fine, I would assume that these would be on the operator controller as well, but if it is just temporary for testing, make sure to add a comment indicating so.